src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+
+function App() {
+  const calculator = {
+    total: '1000',
+    next: '34',
+    operation: 'X',
+  };
```

Comment:
- [ ] In a calculator, the default value is always displayed a 0. Since we are building one, our default value should also be a 0 and not a fixed value. State should be used to change these values. Kindly work on this

Reply:
I created this object for instance. As the requirements say that I have to import calculate module, I also have to use it somehow because of eslint. I will change it to 0 but you should know that this is just an instance to display that method works correctly.